builder/cloudstack/config.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 	Hypervisor        string   `mapstructure:"hypervisor"`
 	InstanceName      string   `mapstructure:"instance_name"`
 	Keypair           string   `mapstructure:"keypair"`
+	TemporaryKeypair  string   `mapstructure:"temporary_keypair"`
```

Comment:
Ah, the docs say it correctly... But that means this is wrong and this should be `TempKeypairName` and `temporary_keypair_name`.